src/universal/components/OutcomeCard/OutcomeCard.js

```diff
@@ -71,17 +73,21 @@ export default class OutcomeCard extends Component {
     const {openMenu} = this.state;
     const {
       content,
-      editingBy,
+      dispatch,
+      editing,
       owner,
+      teamId,
       teamMembers,
       teamMemberId,
       status,
       isArchived,
       isProject,
       updatedAt,
       projectId,
-      handleSubmit
+      handleSubmit,
     } = this.props;
+    const normalizedProjectId = `Task::${projectId}`;
```

Comment:
Not sure yet... Nothing urgent. But, I can see a world where we might want to add Agenda items, checklist item descriptions, metrics, or other optional meeting components we haven't yet dreamed of.

Figured I'd start with general language rather than specific...